src/app/pages/CpsSty/index.jsx

```diff
@@ -90,6 +82,65 @@ const CpsStyContainer = ({ pageData }) => {
     }
   `;
 
+  const StyledSide = styled.div`
```

Comment:
I'm beginning to think we could move this sidebar grid layout to the Grid component within Simorgh, for when other pagetypes require a second column, similar to how the front pages have done it.

https://github.com/bbc/simorgh/blob/44fc8c7f0af9bec25a1e8a3f370bae9aea5385b4/src/app/components/Grid/index.jsx#L50-L59

Reply:
Have just read [your comment](https://github.com/bbc/simorgh/pull/5417#issuecomment-583360968) @HarveyPeachey which basically agrees with this point.